doc/script_commands.txt

```diff
@@ -299,28 +291,25 @@ these floating NPC objects are for. More on that below.
 This will define a shop NPC, which, when triggered (which can only be done by 
 clicking) will cause a shop window to come up. No code whatsoever runs in shop 
 NPCs and you can't change the prices otherwise than by editing the script 
-itself. (No variables even exist at this point of scripting, so don't even 
-bother trying to use them.)
+itself.
 
 The item id is the number of item in the 'item_db.txt' database. If Price is set 
 to -1, the 'buy price' given in the item database will be used. Otherwise, the 
 price you gave will be used for this item, which is how you create differing 
 prices for items in different shops.
 
-Since trunk r12264 you can alternatively use "cashshop" in place of "shop"
-to use the Cash Shop interface, allowing you to buy items with special points
-(Currently stored as account variables in global_reg #CASHPOINTS and #KAFRAPOINTS.)
-This type of shop will not allow you to sell items at it, you may only
-purchase items here. The layout used to define sale items still count, and
+There are other types of shops available:
+cashshop - use "cashshop" in place of "shop" to use the Cash Shop interface, allowing 
+you to buy items with special points that are stored as account variables in `acc_reg_num` 
```

Comment:
Just drop the part where they are stored, because the table name is configurable.